### PR TITLE
Add paymentCardExpiration met on Payplug

### PR DESCRIPTION
**Pull request by Crocmagnon**
This commit intends to add support for `paymentCardExpiration`, I'm not sure anymore if that syntax will work 🤔

### CODE REVIEW

File: src/services/autofill.service.ts [hunk 1]
```diff
@@ -431,7 +431,7 @@ export default class AutofillService implements AutofillServiceInterface {
                 } else if (!fillFields.exp && this.isFieldMatch(f[attr],
                     ['cc-exp', 'card-exp', 'cc-expiration', 'card-expiration', 'cc-ex', 'card-ex',
                         'card-expire', 'card-expiry', 'validite', 'expiration', 'expiry', 'mm-yy',
-                        'mm-yyyy', 'yy-mm', 'yyyy-mm', 'expiration-date'],
+                        'mm-yyyy', 'yy-mm', 'yyyy-mm', 'expiration-date', 'payment-card-expiration'],
                     ['mm-yy', 'mm-yyyy', 'yy-mm', 'yyyy-mm', 'expiration-date'])) {
```
Comment (kspearrin) [inline src/services/autofill.service.ts]: Since the term is pretty specific, you could it to the "contains" array below as well.

Comment (Crocmagnon) [reply src/services/autofill.service.ts]: So should it only appear in the array below or in this one too ?

Comment (kspearrin) [reply src/services/autofill.service.ts]: It should be in both. `mm-yyyy` is an example.

**Comment by Crocmagnon** [comment]
That should cover the request change !

**Pull request merged**